salesagility/SuiteCRM
# FIX #7780 - Bad css format in Date and Date Range Inputs in search forms

## Description




Read #7780

## Motivation and Context


Improve SuiteCRM Date and Date Range Inputs look&feel

### Modal Search

In this example we removed useless free space:

![fecha_fix1](https://user-images.githubusercontent.com/939888/63992211-364c9c00-cac1-11e9-94c0-7d5c2e642c57.png)

![DateSearchModal_Fix](https://user-images.githubusercontent.com/939888/64059037-454b5100-cb8a-11e9-8e2b-7b7093cfcf7e.png)


### Legacy Search


In this example we removed useless free space too and the option selecter fits in the combo.

![fecha_fix2](https://user-images.githubusercontent.com/939888/63992212-38aef600-cac1-11e9-8115-79d2b91f2f7b.png)

![DateSearchForm_Fix](https://user-images.githubusercontent.com/939888/64059041-54320380-cb8a-11e9-8893-f59e89d3846b.png)


## How To Test This


Open modal search in any module (for example Opportunities) and verify in date Range inputs.

To enable legacy search form include this in config_override.php

```php
$sugar_config['enable_legacy_search'] = array(
    'Opportunities',
);
```

## Types of changes


### Final checklist
